### ISSUE TITLE
Bug: Background jobs seems not be running in parallel and db version is different in 2.4.172 (f88f19123587864af127893f119e7118228e5b10)

### ISSUE THREAD

**Issue by eowyn263** [MISP/MISP#9154]
### Expected behavior

1. Showing the same versions of Expected and Actual DB versions (this was tested successfully in 2.4.172 (2ab3245ad44a2bd10aca8f05a1e91b1d4a0e2ad1))
2. Background jobs should help in case of doing heavy performance tasks - the user should be still able to use the platform on a second tab in the same browser (this was tested successfully in 2.4.172 (2ab3245ad44a2bd10aca8f05a1e91b1d4a0e2ad1))

### Actual behavior

I have updated to the newest version 2.4.172 (f88f19123587864af127893f119e7118228e5b10) and implemented the background jobs and I'm seeing the following behavior:

1. Expected DB_version: 113 is different than Actual DB_version: 114
2. Tasks are not running in parallel even with enabled background jobs, e.g. if you run a heavy search, you have to wait till it finishes and only after this you can continue using the platform. While you can still continue working on different browser with a new session, etc., the background job should resolve this

This was not the case in the other instance with 2.4.172 (2ab3245ad44a2bd10aca8f05a1e91b1d4a0e2ad1), note that there is the same version, but a different hash - this one I have updated 3 days ago

### Steps to reproduce

Update to 2.4.172 (f88f19123587864af127893f119e7118228e5b10)

1. Check the diagnostic page and the DB versions
2. Try running some heavy search or adding hundreds of attributes with heavy correlation (for example CIDR correlation) and try to open a new tab in the same browser - this shouldn't wait for the heavy task to finish, but the user should be able to use the platform as normally

### Version

2.4.172 (f88f19123587864af127893f119e7118228e5b10) 

### Operating System

Rhel

### Operating System version

7

### PHP version

7.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adulau** [comment]
I bet it's related to this missing `break` statement 

https://github.com/MISP/MISP/commit/2ab3245ad44a2bd10aca8f05a1e91b1d4a0e2ad1

**Comment by eowyn263** [comment]
Hi @adulau, I just checked the file on the server where I'm seeing this issue and also the one without the issue and the break is in both, so doesn't seem like it :/

I guess it still wouldn't explain the behavior of the background jobs in the version with hash f88f19123587864af127893f119e7118228e5b10

**Comment by iglocska** [comment]
No, nothing to do with the break statement. Simply forgot to dunno the schema after the last hoyfix, please ignore the mismatch for now.

As for the background processing, that's not how it works. Background processing will spin off certain tasks such as emailing, synchronisation etc. Searches are not in scope for it. However, your issues stems from using file sessions, switch to redis sessions and you can multi task.

**Comment by eowyn263** [comment]
Hi @iglocska - that's true, the session setting was incorrect, thanks for the help!
